packages/service-integration/Dockerfile

```diff
@@ -48,6 +48,8 @@ RUN cd packages/service-integration \
 
 # -------------------------- Build stage -------------------
 
+FROM base as development
+# NOTE: this is necessary to allow to build development images but is the same as production here
```

Comment:
i thouught I added this in master ... :-/

Reply:
this might be. I'll double check

Reply:
nope it's not.